From: Onur Özkan <work@onurozkan.dev>
Subject: checkpatch: document new check PLACEHOLDER_USE
Date: Wed, 17 Sep 2025 20:37:25 +0300

Adds documentation for the new check PLACEHOLDER_USE in checkpatch.

Link: https://lkml.kernel.org/r/20250917173725.22547-3-work@onurozkan.dev
Signed-off-by: Onur Özkan <work@onurozkan.dev>
Acked-by: Joe Perches <joe@perches.com>
Cc: Andy Whitcroft <apw@canonical.com>
Cc: Dwaipayan Ray <dwaipayanray1@gmail.com>
Cc: Jonathan Corbet <corbet@lwn.net>
Cc: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/dev-tools/checkpatch.rst |   10 ++++++++++
 1 file changed, 10 insertions(+)

--- a/Documentation/dev-tools/checkpatch.rst~checkpatch-document-new-check-placeholder_use
+++ a/Documentation/dev-tools/checkpatch.rst
@@ -1245,6 +1245,16 @@ Others
     The patch file does not appear to be in unified-diff format.  Please
     regenerate the patch file before sending it to the maintainer.
 
+  **PLACEHOLDER_USE**
+    Detects unhandled placeholder text left in cover letters or commit headers/logs.
+    Common placeholders include lines like::
+
+      *** SUBJECT HERE ***
+      *** BLURB HERE ***
+
+    These typically come from autogenerated templates. Replace them with a proper
+    subject and description before sending.
+
   **PRINTF_0XDECIMAL**
     Prefixing 0x with decimal output is defective and should be corrected.
 
_
